pyodide-build/pyodide_build/tests/fixture.py

```diff
@@ -88,7 +86,7 @@ def xbuildenv(selenium, tmp_path, reset_env_vars, reset_cache):
 
     assert result.returncode == 0
 
-    version_dir = envpath / __version__
+    version_dir = envpath / "temp_version"
```

Comment:
Where does `temp_version` come from?

Reply:
It is a random name to test modifications to this PR. Before this PR, if there is no folder with a name correspond to the `__version__` (e.g. 0.26.0), it assumed that xbuildenv was not installed and downloaded that version remotely.

So to test this, I renamed the xbuildenv folder to something random other than __version__ to see if it work.